Formula/opencascade.rb

```diff
@@ -0,0 +1,45 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+
+  depends_on "cmake" => :build
+  depends_on "doxygen" => :build
+  depends_on "freeimage"
+  depends_on "freetype"
+  depends_on "gl2ps"
+  depends_on "tbb"
+
+  def install
+    sdk = MacOS::CLT.installed? ? "" : MacOS.sdk_path
+    system "cmake", ".",
+             "-DUSE_GL2PS=ON",
+             "-DUSE_FREEIMAGE=ON",
```

Comment:
with the rest aligned beneath

Reply:
If I start on the same line and use the same indent, some lines exceed 80 characters?  Which is it?

Reply:
The former. There has to be room to doodle a snowman in the margin on the left.

Reply:
Done - doodle away ;)